ogr/ogrsf_frmts/miramon/ogrmiramonlayer.cpp

```diff
@@ -0,0 +1,2613 @@
+                {
+                    if (phMiraMonLayer->iMultiRecord == -1)
+                        GoToFieldOfMultipleRecord(
+                            nIElem,
+                            phMiraMonLayer->pMultRecordIndex[nIElem].nMR - 1,
+                            nIField);
+                    else if ((MM_EXT_DBF_N_MULTIPLE_RECORDS)
+                                 phMiraMonLayer->iMultiRecord <
+                             phMiraMonLayer->pMultRecordIndex[nIElem].nMR)
+                        GoToFieldOfMultipleRecord(
+                            nIElem,
+                            (MM_EXT_DBF_N_MULTIPLE_RECORDS)
+                                phMiraMonLayer->iMultiRecord,
+                            nIField);
+                    else
+                    {
+                        memset(phMiraMonLayer->szStringToOperate, 0,
+                               phMiraMonLayer->pMMBDXP->pField[nIField]
+                                   .BytesPerField);
+                        continue;
+                    }
+                }
+                else
+                    GoToFieldOfMultipleRecord(nIElem, 0, nIField);
+
+                memset(phMiraMonLayer->szStringToOperate, 0,
+                       phMiraMonLayer->pMMBDXP->pField[nIField].BytesPerField);
+                fread_function(
+                    phMiraMonLayer->szStringToOperate,
+                    phMiraMonLayer->pMMBDXP->pField[nIField].BytesPerField, 1,
+                    phMiraMonLayer->pMMBDXP->pfDataBase);
+                phMiraMonLayer
+                    ->szStringToOperate[phMiraMonLayer->pMMBDXP->pField[nIField]
+                                            .BytesPerField] = '\0';
+
+                MM_RemoveWhitespacesFromEndOfString(
+                    phMiraMonLayer->szStringToOperate);
+                if (!MMIsEmptyString(phMiraMonLayer->szStringToOperate))
+                {
+                    char pszDate_5[5];
+                    char pszDate_3[3];
+                    int Year, Month, Day;
+
+                    strncpy(pszDate_5, phMiraMonLayer->szStringToOperate, 4);
+                    pszDate_5[4] = '\0';
+                    Year = atoi(pszDate_5);
+
+                    strncpy(pszDate_3, phMiraMonLayer->szStringToOperate + 4,
+                            2);
+                    (pszDate_3)[2] = '\0';
+                    Month = atoi(pszDate_3);
+
+                    strncpy(pszDate_3, phMiraMonLayer->szStringToOperate + 6,
+                            2);
+                    (pszDate_3)[2] = '\0';
+                    Day = atoi(pszDate_3);
+
+                    poFeature->SetField(nIField, Year, Month, Day);
+                }
+                else
+                    poFeature->SetField(nIField,
+                                        phMiraMonLayer->szStringToOperate);
+            }
+        }
+    }
+
+    poFeature->SetFID(nFeatureId);
+
+    // Perhaps there are features without geometry.
```

Comment:
doesn't look like that comment is relevant, at least here
```suggestion
```

Reply:
Deleted